Development-Guide/Git-Guides-and-Standards/Commit-Messages.adoc

```diff
@@ -19,28 +19,33 @@ Further paragraphs come after blank lines.
 - Bullet points are okay, too.
 - Typically a hyphen or asterisk is used for the bullet, followed by a
   single space. Use a hanging indent.
-```
-
-Your commit title should be phrased so it can answer the question "If this commit is applied, it will..."
-
-### Example commit message
 
-```
-Change the units of cell voltages from mV to V
-
-Based on testing and feedback....
+Fixes/Continues <Your-JIRA-Ticket>
 ```
 
 #### Subject line
 
-The subject line should be able to describe the effects of the patch:
+The subject line should be able to answer the question:
```

Comment:
Might not be a bad idea to add the Jira issue to the PR subject as well (i.e. `SFT-2 Update git standards for clarity & instructions for JIRA integration`)
Up to you though

Reply:
I like it